docs/oauth2.md

```diff
@@ -62,149 +71,81 @@ auth_backends.1 = rabbit_auth_backend_oauth2
-| `auth_oauth2.jwks_url`                   | The URL of key server. According to the [JWT Specification](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.2) key server URL must be https.
-| `auth_oauth2.https.cacertfile`           | Path to a file containing PEM-encoded CA certificates. The CA certificates are used during key server [peer verification](./ssl#peer-verification).
-| `auth_oauth2.https.depth`                | The maximum number of non-self-issued intermediate certificates that may follow the peer certificate in a valid [certification path](./ssl#peer-verification-depth). Default is 10.
-| `auth_oauth2.https.peer_verification`    | Should [peer verification](./ssl#peer-verification) be enabled. Available values: `verify_none`, `verify_peer`. Default is `verify_none`. It is recommended to configure `verify_peer`. Peer verification requires a certain amount of setup and is more secure.
-| `auth_oauth2.https.fail_if_no_peer_cert` | Used together with `auth_oauth2.https.peer_verification = verify_peer`. When set to `true`, TLS connection will be rejected if client fails to provide a certificate. Default is `false`.
-| `auth_oauth2.https.hostname_verification`| Enable wildcard-aware hostname verification for key server. Available values: `wildcard`, `none`. Default is `none`.
-| `auth_oauth2.algorithms`                 | Restrict [the usable algorithms](https://github.com/potatosalad/erlang-jose#algorithm-support).
-| `auth_oauth2.verify_aud`                 | [Verify token's `aud`](#token-validation).
-
-For example:
-
-Configure with key files
+| Key                                        | Documentation
+|--------------------------------------------|-----------
+| `auth_oauth2.resource_server_id`           | [The Resource Server ID](#resource-server-id)
+| `auth_oauth2.resource_server_type`         | [The Resource Server Type](#rich-authorization-request)
+| `auth_oauth2.additional_scopes_key`        | Configure the plugin to also look in other fields (maps to `additional_rabbitmq_scopes` in the old format). |
+| `auth_oauth2.scope_prefix`                 | Configure prefix for all scopes. Default value is  `auth_oauth2.resource_server_id` followed by the dot `.` character. |
+| `auth_oauth2.preferred_username_claims`    | List of JWT claims to look for username associated to the token separated by commas.
+| `auth_oauth2.default_key`                  | ID of the default signing key.
+| `auth_oauth2.signing_keys`                 | Paths to signing key files.
+| `auth_oauth2.issuer`                       | The issuer URL of authorization server used to discover endpoints such as `jwk_uri` and others (https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata).
+| `auth_oauth2.jwks_url`                     | The URL of key server. According to the [JWT Specification](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.2) key server URL must be https.
```

Comment:
Add "the" after "URL of"
Add comma after "JWT specification" and then add "the" before "key"